### ISSUE TITLE
Feature Request: Support Jest expectations with Mocha

### ISSUE THREAD

**Issue by tomitrescak** [wallabyjs/public#2445]
Hi, thanks for the amazing work.
I have a bit odd request.

I am using `mocha` with jest matchers (package [expect](https://jestjs.io/docs/en/expect.html)).
With this combination, Wallaby does not process nicely the output from the "expect" library. I really love the "compare side-by-side" feature when using `chai` in mocha or `expect` in jest.

Would you consider parsing the output of "expect" library even in mocha?
Currently, all I get is the standard +/- output.

Thanks!

FYI: This is what Wallaby shows now for Equality

```
expect(received).toEqual(expected) // deep equality 
     
    - Expected  - 1 
    + Received  + 2 
     
      Object { 
        "a": 1, 
    -   "b": 3, 
    +   "b": 2, 
    +   "c": 3, 
      } 
```

While with running something similar in jest I receive:

```
 ​Show side-by-side diff​
    
    { a: 1, b: ‌⁠2, c: ‌⁠3 }
     
    - Expected  - 1 
    + Received  + 2 
     
      Object { 
        "a": 1, 
    -   "b": 3, 
    +   "b": 2, 
    +   "c": 3, 
      } 
```

It's just that expect is so much nicer to use than chai

**Comment by ArtemGovorov** [comment]
Hey @tomitrescak, we've just implemented and published the feature in [the latest core v1.0.909](https://wallabyjs.com/docs/intro/troubleshooting.html#forcing-wallabyjs-core-update).

**Comment by tomitrescak** [comment]
Aahahahahahaha! You are soooo amazing! 
I spent 3 days trying to build a library that would remap all commands from `expect` to `chai`. This is too good! The best investment since 2016 ;)
